adp-aso-helm-library/templates/_storage-account.yaml

```diff
@@ -23,7 +24,7 @@ spec:
   accessTier: {{ $storageAccount.accessTier | default "Hot" }}
   kind: StorageV2
   sku:
-    {{- if ne $environment "snd" }}
+    {{- if eq $enablePrivateEndpoint "true" }}
```

Comment:
SKU should not be derived from 'enablePrivateEndpoint' value. Though it will work in this scenario, should we use diff variable?

Reply:
ok, I will look into it.